aloscene/renderer/renderer.py

```diff
@@ -93,12 +93,14 @@ def render(self, method="matplotlib", location: str = None, figsize=[6.4, 4.8]):
         if location is not None:
             plt.figure(figsize=figsize, tight_layout=True)
             plt.imshow(self.image)
+            plt.title("Frame" if self.title is None else self.title, self.image[:, :, ::-1])
```

Comment:
"Frame" can not be a view title, I prefer to call `plt.title` only if self.title is not None to keep the previous behavior.

Reply:
Why not ? It is with the OpenCV viewer (https://github.com/Visual-Behavior/aloception-oss/blob/master/aloscene/renderer/renderer.py#L106) so I was trying to have the same behavior for both viewers